openpype/hosts/max/plugins/publish/validate_loaded_plugin.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+"""Validator for Loaded Plugin."""
+import os
+from pyblish.api import InstancePlugin, ValidatorOrder
+from pymxs import runtime as rt
+
+from openpype.pipeline.publish import (
+    RepairAction,
+    OptionalPyblishPluginMixin,
+    PublishValidationError
+)
+from openpype.hosts.max.api.lib import get_plugins
+
+
+class ValidateLoadedPlugin(OptionalPyblishPluginMixin,
+                           InstancePlugin):
+    """Validates if the specific plugin is loaded in 3ds max.
+    Studio Admin(s) can add the plugins they want to check in validation
+    via studio defined project settings"""
+
+    order = ValidatorOrder
+    hosts = ["max"]
+    label = "Validate Loaded Plugins"
+    optional = True
+    actions = [RepairAction]
+
+    def get_invalid(self, instance):
+        """Plugin entry point."""
+        if not self.is_active(instance.data):
+            self.log.debug("Skipping Validate Loaded Plugin...")
+            return
+
+        required_plugins = (
+            instance.context.data["project_settings"]["max"]["publish"]
+                                 ["ValidateLoadedPlugin"]
+                                 ["family_plugins_mapping"]
+        )
+
+        if not required_plugins:
+            return
+
+        invalid = []
+
+        # get all DLL loaded plugins in Max and their plugin index
+        available_plugins = {
+            plugin_name.lower(): index for index, plugin_name in enumerate(
+                get_plugins())
+        }
+
+        for families, plugin in required_plugins.items():
+            families_list = families.split(",")
+            excluded_families = [family for family in families_list
+                                 if instance.data["family"] != family
+                                 and family != "_"]
```

Comment:
Do we only care about the main `family` or also `families`?

Also, what does `_` equal? Why is that value special? It seems you are using it as a wildcard - which all other plugins use `*` for; why don't match that?

You seem to be collecting the excluded families even though you want to match **included** families right? If it matches any of the families, then include it?

```python
# Out of for loop build the instance family lookup
instance_families = {instance.data["family"]}
instance_families.update(instance.data.get("families", []))

# In the for loop check whether any family matches
match_families = {fam.strip() for fam in families.split(",") if fam.strip()}
has_match = "*" in match_families or match_families.intersection(instance_families)
if not has_match:
    continue
```

Reply:
it's already my bad for the bad variable name.(Need some coffee time)

Reply:
`_` was in my example just because it is in the settings as key in and it doesn't allow `*` and I was lazy to change dict model to samething else. Anyway, we need to check `family` too.

Reply:
> was in my example just because it is in the settings as key in and it doesn't allow * 

How come settings don't allow the `*` in the key - it's just a string, right?

Reply:
maybe we can mention in either docstring or docs (or both) that * allows the validation check on all plugins.